package.json

```diff
@@ -58,6 +58,7 @@
         "exec-sh": "^0.4.0",
         "jest": "^28.1.1",
         "jest-environment-jsdom": "^28.1.1",
+        "jest-extended": "^3.0.1",
```

Comment:
This is a fairly popular set of jest custom matches. https://www.npmjs.com/package/jest-extended